tests/testthat/test-transform-runner.R

```diff
@@ -61,3 +61,30 @@ testthat::test_that("verification passes if data adheres to schema", {
   colnames(transformed_data$people) <- c("name", "age", "height")
   expect_silent(verify_data(con, transformed_data))
 })
+
+context("load_runner")
```

Comment:
one context per file - please move to another one (some versions of test that have had odd behaviour with this)

Reply:
I've removed that line, I think probably a copy + paste error which should never have been there.